onthequad/onthequad/EventEndPoints.py

```diff
@@ -39,9 +39,17 @@ def on_post(self, req, resp):
             params = req.params
             if 'userid' in params:
                 try:
-                    eventData = Event(data['name'], data['location'], data['date'], data['startTime'], data['endTime'],
-                                      data['category'], data['creator'], data['availableSpots'], data['description'],
-                                      data['coordinates'], data['public'])
+                    eventData = Event(data['name'],
+                                      data['location'],
+                                      data['date'],
+                                      data['startTime'],
+                                      data['endTime'],
+                                      data['category'],
+                                      data['creator'],
+                                      data['availableSpots'],
+                                      data['description'],
+                                      data['coordinates'],
+                                      data['public'])
                 except:
                     send = {"msg": "invalid fields"}
                     resp.body = json.dumps(send)
```

Comment:
We need to set a response status here

Reply:
It looks to me like we are setting the status in every spot.

Reply:
The exception block doesn't have a `resp.status = falcon.HTTP_<status>`. At least that's what it looks like to me

Reply:
<img width="933" alt="Screen Shot 2019-04-10 at 3 29 58 PM" src="https://user-images.githubusercontent.com/30120889/55911268-e0c13280-5ba5-11e9-97f7-bf32fcb45488.png">

Reply:
see at the bottom? @paugajt